internal/setup/warning_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package setup
+
+import (
+	"bytes"
+	"os"
+	"strings"
+	"testing"
+
+	"sigs.k8s.io/krew/internal/environment"
+	"sigs.k8s.io/krew/internal/testutil"
+)
+
+func TestCheckSetup_firstRun(t *testing.T) {
```

Comment:
rm test func names :)

Reply:
I guess you meant the forgotten rename?

Reply:
I don't know what I was saying above.
I think I meant make sure Test names are the same as func name.